packages/elements/src/tab/index.ts

```diff
@@ -52,6 +55,13 @@ export class Tab extends ControlElement {
     `;
   }
 
+  /**
+   * Aria indicating current select state
+   * @ignore
+   */
+  @property({ type: String, reflect: true, attribute: 'aria-selected' })
+  public ariaSelected = 'false';
```

Comment:
@Sarin-Udompanish Can we use `this.setAttribute('aria-selected', ... )` instead. I think we have come to an agreement that we do prefer it that way rather than declaring a new Lit property.

Reply:
Fixed

Reply:
So we can safely remove this line yeah ?

Reply:
Yes, removed.